imgparse/xmp.py

```diff
@@ -41,25 +41,39 @@ class SensorMake(NamedTuple):
     """
 
     RELATIVE_ALT: re.Pattern
+    LRF_ALT: re.Pattern
     ROLL: re.Pattern
     PITCH: re.Pattern
     YAW: re.Pattern
+    FOCAL_LEN: re.Pattern
 
 
 Sentera = SensorMake(
     RELATIVE_ALT=re.compile(r"Camera:AboveGroundAltitude[^-\d]*(-?[0-9]+.[0-9]+)"),
+    LRF_ALT=re.compile(
+        r"Sentera:AltimeterCalcul?atedAGL[^-\d]*(-?[0-9]+.[0-9]+)"
+    ),  # l was left out in Quad v1.0.0
     ROLL=re.compile(r"Camera:Roll[^-\d]*(-?[0-9]+.[0-9]+(?:E-?[0-9]+)?)"),
     PITCH=re.compile(r"Camera:Pitch[^-\d]*(-?[0-9]+.[0-9]+(?:E-?[0-9]+)?)"),
     YAW=re.compile(r"Camera:Yaw[^-\d]*(-?[0-9]+.[0-9]+(?:E-?[0-9]+)?)"),
+    FOCAL_LEN=re.compile(r"Camera:PerspectiveFocalLength[^-\d]*(-?[0-9]+.[0-9]+)"),
 )
 
 DJI = SensorMake(
-    RELATIVE_ALT=re.compile(r'drone-dji:RelativeAltitude="(-?\+?[0-9]+.[0-9]+)"'),
-    ROLL=re.compile(r'drone-dji:GimbalRollDegree="(-?\+?[0-9]+.[0-9]+(?:E-?[0-9]+)?)"'),
+    RELATIVE_ALT=re.compile(r"drone-dji:RelativeAltitude[^-\d]*(-?\+?[0-9]+.[0-9]+)"),
+    LRF_ALT=re.compile(r"Only supported for Sentera sensors"),
```

Comment:
Is this necessary?

Reply:
If you edit the XMP with exiftool, it changes the formatting somewhat. This regex accepts either format.

Reply:
I could be wrong, but I think Elliott was referring to adding the `LRF_ALT` tag to the `DJI` `SensorMake` (if he wasn't, that's the part I'm curious about at least). You can just set that as `None` if you don't want to add a regex for it, or you can make the field optional in the class definition by setting its default to `None`.

Reply:
I've changed it to None